src/osd/PrimaryLogPG.cc

```diff
@@ -10466,6 +10465,7 @@ void PrimaryLogPG::mark_all_unfound_lost(
 	} // otherwise, just do what we used to do
 	dout(10) << e << dendl;
 	log_entries.push_back(e);
+	missing_loc.recovered(oid);
```

Comment:
Unfortunately, I'd need to have all the LOST_DELETE deleted OIDs available to the callback function.